react-app/src/redux/Auth/reducer.js

```diff
@@ -4,10 +4,15 @@ import { loadState } from '../localStorage';
 
 import { actionNames } from './actions';
 
-const defaultInitialState = Immutable({
+const token = loadState('token', '');
+const tokenExpireDateTime = loadState('tokenExpireDateTime', '');
+const logged = token && new Date().getTime() <= tokenExpireDateTime && true;
```

Comment:
You can do this instead:
`const logged = token && !!(new Date().getTime() <= tokenExpireDateTime);`